src/commands/rubocop-check.yml

```diff
@@ -0,0 +1,16 @@
+description: "Check the code by Rubocop. You have to add `gem 'rubocop'` to your Gemfile. Enable parallelism on CircleCI for faster checking."
+parameters:
+  format:
+    description: "Customize the formatter for rubocop https://docs.rubocop.org/rubocop/0.88/formatters.html"
+    default: "progress"
+    type: string
+
+  out-path:
+    description: "Customize the directory of output file"
+    default: "/tmp/rubocop-results"
+    type: string
+
+steps:
+  - run: |
+      mkdir -p <<parameters.out-path>>
+      bundle exec rubocop --out <<parameters.out-path>>/check-results.xml --format <<parameters.format>>
```

Comment:
We'll have to ensure rubocop is installed before executing this :)

Reply:
I think this can be decided by the users. If they don't need the function from rubocop(isn't installed), they will not add `ruby/rubocop-check` command in CI steps. I have checked the code about rspec

```
    - run: |
        mkdir -p <<parameters.out-path>>
        TESTFILES=$(circleci tests glob "spec/**/*_spec.rb" | circleci tests split --split-by=timings)
        bundle exec rspec $TESTFILES --profile 10 --format RspecJunitFormatter --out <<parameters.out-path>>/results.xml --format progress
```

It doesn't check if the rspec is installed, before executing it. So I think that rubocop can apply the same strategy.

Reply:
Fair enough!